### PR TITLE
Add dev configuration files

### PR DESCRIPTION
**Pull request by sei-bstein**
- Add a dev configuration to support future developers on the project

### CODE REVIEW

**Comment by sei-jmattson** [comment]
Adding `assets/settings.json` and `environment.Development.json` seems unnecessary. Creates friction for dev's who alter them..having to ensure they don't get committed back.  IMO, this is better suited to a *Dev Section* in the Readme.

**Comment by sei-bstein** [comment]
> Adding `assets/settings.json` and `environment.Development.json` seems unnecessary. Creates friction for dev's who alter them..having to ensure they don't get committed back. IMO, this is better suited to a _Dev Section_ in the Readme.

Committing `settings.json` was inadvertent - I'm rolling that part back. 

The context here is that @sei-mkaar and I have invested some not-insignificant time in creating a starter dev setup that allows devs who don't touch Topo or Identity to have access to those applications without needing to have full copies of their source locally. Matt created a version of the Foundry appliance which can be deployed via a production Topomojo app and contains running versions of GB, Topo, and Identity. Devs who work on any of those applications, or applications which depend on them, can pull down a copy and run source locally for the app they _are_ working on without worrying about configuration or implementation details of the others. For example, most of my work is on Gameboard, so I run a personal version of Topo and Identity on a VM in our launchpad TM app and connect to it from the version of Gameboard that I'm debugging on my local hardware.

In short, we envision this configuration as a default startup one for onboarding new developers who can later customize their setup when they're more comfortable. If we find maintaining the dev config cumbersome, we can always gitignore it while leaving the version that's in the repo there. Let me know if you have any questions.

**Comment by sei-bstein** [comment]
Update: pulling this per discussion with @sei-mkaar

**Comment by sei-mkaar** [comment]
A readme or doc section sounds like a good idea. The integration with the appliance is non-trivial, and I imagine certain developers might want to go in a different direction with their setup.
